util/feeding.py

```diff
@@ -14,8 +14,9 @@
 
 from util.config import Config
 from util.text import text_to_char_array
+from pertub import speed_pertub, pertub
 
-
+is_pertub = False
```

Comment:
This should rather be exposed as a flag in `utils/flags.py`, defaulting to false as here.

Reply:
Done